cloud-agent/service/server/src/main/scala/org/hyperledger/identus/oid4vci/domain/Openid4VCIProofJwtOps.scala

```diff
@@ -0,0 +1,66 @@
+package org.hyperledger.identus.oid4vci.domain
+
+import com.nimbusds.jose.{JOSEObjectType, JWSAlgorithm, JWSHeader, JWSObject, JWSSigner, Payload}
+import org.hyperledger.identus.castor.core.model.did.LongFormPrismDID
+import org.hyperledger.identus.pollux.vc.jwt.{DidResolver, JWT}
+import org.hyperledger.identus.pollux.vc.jwt.JwtSignerImplicits.*
+import org.hyperledger.identus.shared.crypto.Secp256k1PrivateKey
+import zio.Task
+import zio.ZIO
+
+import java.util.UUID
+import scala.jdk.CollectionConverters.*
+import scala.util.Try
+
+trait Openid4VCIProofJwtOps {
+
+  val jwtTypeName = "openid4vci-proof+jwt"
+
+  val supportedAlgorithms: Set[String] = Set("ES256K")
+
+  private def buildHeaderFromLongFormDid(longFormDID: LongFormPrismDID) = {
+    new JWSHeader.Builder(JWSAlgorithm.ES256K)
+      .keyID(longFormDID.toString)
```

Comment:
Are we missing the ID fragment in `kid`? like in `did:prism:....#key-1`

Reply:
I will fix it in the next PR